drape_frontend/apply_feature_functors.cpp

```diff
@@ -777,7 +777,7 @@ void ApplyLineFeatureGeometry::operator() (m2::PointD const & point)
   {
     if (m_simplify &&
         ((m_spline->GetSize() > 1 && point.SquaredLength(m_lastAddedPoint) < m_minSegmentSqrLength) ||
-        m_spline->IsPrelonging(point)))
+          m_spline->IsProlonging(point)))
```

Comment:
That looks worse than another approach:
```
    if (m_simplify
        && ((m_spline->GetSize() > 1 && point.SquaredLength(m_lastAddedPoint) < m_minSegmentSqrLength)
        || m_spline->IsProlonging(point)))
```
Agree?

Reply:
I don't like it, and this is confusing: && and II here are not in one level.

Reply:
Then something like this:
```
if (m_simplify
        && ((m_spline->GetSize() > 1 && point.SquaredLength(m_lastAddedPoint) < m_minSegmentSqrLength)
                || m_spline->IsProlonging(point)))
```
I don't like it either :)